ci-operator/step-registry/osd-ccs/cluster/provision/aws/osd-ccs-cluster-provision-aws-commands.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+subfix=$(head /dev/urandom | tr -dc a-z0-9 | head -c 4)
+CLUSTER_NAME=${CLUSTER_NAME:-"ci-osd-ccs-$subfix"}
```

Comment:
@yasun1 as we talked, we have two label OSD and CCS. how do we check if it is osd or ccs cluster per this cluster name? thanks

Reply:
`osd-ccs` is the osd ccs cluster. `osd` will be the osd non-ccs cluster.